http://bugs.gentoo.org/468160

From: Jaroslav Kysela <perex@perex.cz>
Date: Mon, 15 Apr 2013 12:44:13 +0000 (+0200)
Subject: alsactl: Fix the string size for the lock file contents
X-Git-Url: http://git.alsa-project.org/?p=alsa-utils.git;a=commitdiff_plain;h=95788fea25c1a59985828d4b91af0772d077600b

alsactl: Fix the string size for the lock file contents

The string length is 10 characters + LF + '\0' = 12 characters.

Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---

diff --git a/alsactl/lock.c b/alsactl/lock.c
index d34d013..fce208b 100644
--- a/alsactl/lock.c
+++ b/alsactl/lock.c
@@ -35,7 +35,7 @@ static int state_lock_(const char *file, int lock, int timeout)
 	int fd = -1, err = 0;
 	struct flock lck;
 	struct stat st;
-	char lcktxt[11];
+	char lcktxt[12];
 	char *nfile;
 
 	if (!do_lock)
